ipaserver/install/server/upgrade.py

```diff
@@ -1635,6 +1635,19 @@ def disable_httpd_system_trust(http):
             db.add_cert(cert, nickname, trust_flags)
 
 
+def update_replica_config(db_suffix):
+    dn = DN(
+        ('cn', 'replica'), ('cn', db_suffix), ('cn', 'mapping tree'),
+        ('cn', 'config')
+    )
+    entry = api.Backend.ldap2.get_entry(dn)
```

Comment:
When the topology contains a single master, the cn=replica entry does not exist. The code should catch the exception NotFound, otherwise upgrade will fail.

Reply:
I've got a PR forthcoming to fix this on master.

Reply:
PR: https://github.com/freeipa/freeipa/pull/1812